SDL/Event.cu

```diff
@@ -725,8 +725,13 @@ __global__ void addPixelSegmentToEventKernel(unsigned int* hitIndices0,unsigned
       unsigned int outerMDIndex = rangesInGPU.miniDoubletModuleIndices[pixelModuleIndex] + 2*(tid) +1;
       unsigned int pixelSegmentIndex = rangesInGPU.segmentModuleIndices[pixelModuleIndex] + tid;
 
+//#ifdef CUT_VALUE_DEBUG
+//      addMDToMemory(mdsInGPU, hitsInGPU, modulesInGPU, hitIndices0[tid], hitIndices1[tid], pixelModuleIndex, 0,0,0,0,0,0,0,0,0,0,0,0,0,innerMDIndex);
+//      addMDToMemory(mdsInGPU, hitsInGPU, modulesInGPU, hitIndices2[tid], hitIndices3[tid], pixelModuleIndex, 0,0,0,0,0,0,0,0,0,0,0,0,0,outerMDIndex);
+//#else
```

Comment:
Why are these commented out? Are we to drop them or is this a leftover from a test?

Reply:
yes, this not used anywhere :)